pydis_site/apps/events/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, re_path
+
+from pydis_site.apps.events.views import IndexView, PageView
+
+app_name = "events"
+urlpatterns = [
+    path("", IndexView.as_view(), name="events"),
+    re_path("(?P<path>.+)/$", PageView.as_view(), name="page"),
```

Comment:
```suggestion
    path("<str:path>/", PageView.as_view(), name="page"),
```
You don't need to use `re_path` here, you can use `path`'s [builtin captured values.](https://docs.djangoproject.com/en/3.1/topics/http/urls/#example)